roles/docker/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-# defaults for mirsg.docker
+docker_version: 24.0.9
```

Comment:
sad we need to specify twice :(

Reply:
at least they're different versions now 😄